packages/ui/src/ui/pages/query-tracker/Navigation/api/loadSyncTableRequest.ts

```diff
@@ -0,0 +1,77 @@
+import {wrapApiPromiseByToaster} from '../../../../utils/utils';
+import {YTApiId, ytApiV3Id} from '../../../../rum/rum-wrap-api';
+import {
+    parseErrorFromResponse,
+    prepareHeaders,
+    prepareRows,
+} from '../../../../utils/navigation/content/table/table';
+import {injectColumnsFromSchema} from '../../../../utils/navigation/content/table/table-ts';
+import {NavigationTableSchema} from '../../module/queryNavigation/queryNavigationSlice';
+import {UnipikaValue} from '../../../../components/Yson/StructuredYson/StructuredYsonTypes';
+import {getProxyPath} from './getProxyPath';
+import {JSONParser} from '../../module/api';
+import {TypeArray} from '../../../../components/SchemaDataType/dataTypes';
+
+type LoadStaticTableRows = (props: {
+    path: string;
+    cluster: string;
+    schema: NavigationTableSchema[];
+    limit: number;
+    output_format: Record<string, any> | string;
+}) => Promise<{columns: string[]; rows: UnipikaValue[]; yqlTypes: TypeArray[] | null}>;
+
+export const loadSyncTableRequest: LoadStaticTableRows = async ({
```

Comment:
`Sync` should be replaced with `Static`
What is the reason to have `Request` suffix in the name?
Is it possible to have reusable function with/from Navigation?

Reply:
I added the suffix to indicate that this function is not an action, but rather executes a request and processes the result.

The fact is that both implementations differ in the parameters of the ytApiV3Id.readTable function. For side navigation, it is somewhat simpler